workers/waitgroup.go

```diff
@@ -0,0 +1,39 @@
+package workers
+
+import (
+	"runtime"
+
+	"go.uber.org/atomic"
+)
+
+type WaitGroup interface {
```

Comment:
mb `Waiter`?

Reply:
ok

Reply:
but I think it's better to be ```WaitGroup``` as alternative sync.WaitGroup